pkg/cmd/roachtest/spec/option.go

```diff
@@ -248,17 +262,15 @@ func GCEMinCPUPlatform(platform string) Option {
 }
 
 // GCEVolumeType sets the volume type when the cluster is on GCE.
+// Deprecated: use VolumeType instead.
 func GCEVolumeType(volumeType string) Option {
```

Comment:
nit: I see 6 tests that use this deprecated option, can we just get rid of it now?